lib/dockerize_rails/docker_commands/docker_build.rb

```diff
@@ -60,13 +78,26 @@ def self.build_postgres
       end
 
       def self.build_docker_image(dockerfile, repo)
-        image = Docker::Image.build_from_dir(
-          '.',
-          dockerfile: dockerfile
-        )
+        image = Docker::Image.build_from_dir('.', dockerfile: dockerfile) do |v|
+          if DRNameSpace.namespace.stream_log
+            if (log = JSON.parse(v)) && log.has_key?('stream')
```

Comment:
__[rubocop]__
* Use `Hash#key?` instead of `Hash#has_key?`. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-key)

<!-- Comment by [SideCI](https://sideci.com) -->